docassemble/juvenilesealing/data/questions/juvenile-sealing-informational-text.yml

```diff
@@ -25,9 +25,9 @@ template: more_form_info
 content: |
   ## More about this form
   
-  <span class="new">If you have the information you need,</span>
+  If you have the information you need,
   these questions
-  can take about to **40 minutes** to answer. ${records_can_help}
+  can take upto **40 minutes** to answer. ${records_can_help}
```

Comment:
Good attempt, but this needs a space! It's also weird because I thought I fixed that...

Reply:
Ah, I changed this in copy of docket sheets v5, not in here. Damnit. I built this poorly. See https://github.com/knod/docassemble-juvenilesealing/commit/178ed7a0343659fa8d7151bdbc0499ffb6c4dcf9 and tell me if it looks ok.

Reply:
A lot of this file was for v7. I'm going to try to clear this up as soon as I can get something unbroken.